airflow/www/extensions/init_views.py

```diff
@@ -267,24 +271,30 @@ def init_api_connexion(app: Flask) -> None:
     base_path = "/api/v1"
     base_paths.append(base_path)
 
-    with ROOT_APP_DIR.joinpath("api_connexion", "openapi", "v1.yaml").open() as f:
-        specification = safe_load(f)
```

Comment:
The specification was preloaded for some speedup in application startup (see #29631)